src/librustc_metadata/rmeta/encoder.rs

```diff
@@ -1612,13 +1612,17 @@ impl EncodeContext<'tcx> {
                         EntryKind::TypeParam,
                         default.is_some(),
                     );
+                    if default.is_some() {
+                        self.encode_stability(def_id.to_def_id());
+                    }
                 }
                 GenericParamKind::Const { .. } => {
                     self.encode_info_for_generic_param(
                         def_id.to_def_id(),
                         EntryKind::ConstParam,
                         true,
                     );
+                    // FIXME(const_generics:defaults)
```

Comment:
Please expand the comment.

Reply:
This is only once relevant #44580 Implement defaults for const parameters (FIXME(const_generics:defaults)) is checked. How would you suggest expanding the comment.

Reply:
We've used this comment through, just as an indicator that we're not handling const defaults here, but this is a relevant location when defaults are added.